generate.js

```diff
@@ -78,8 +82,14 @@ const template = `
         </div>
       {{/orgs}}
     </div>
+    <script>
+      ${script}
+    </script>
   </body>
 </html>
 `
 
-fs.writeFileSync('out/index.html', Mustache.render(template, { orgs }))
+fs.writeFileSync(
+  'out/index.html', 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpsmvkva_5/generate.js
+++ b/tmp/tmpsmvkva_5/generate.js
@@ -90,6 +90,6 @@
 `
 
 fs.writeFileSync(
-  'out/index.html', 
+  'out/index.html',
   Mustache.render(template, { orgs, timestamp })
 )
```